packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -1719,6 +1719,28 @@ describe('KeyringController', () => {
           );
         });
       });
+
+      it('should throw error if contract address is not provided', async () => {
+        await withController(async ({ controller, initialState }) => {
+          const account = initialState.keyrings[0].accounts[0];
+
+          for (const invalidContractAddress of [
+            undefined,
+            null,
+          ] as unknown as string[]) {
```

Comment:
Very very minor: Could have been a `it.each` so that if one of the 2 values results in a different behavior, the 2 tests would have been isolated from each other (I don't expect this to change anytime soon, but just wanted to share just in case).